src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -182,7 +183,7 @@ class ContentSidebar extends PureComponent<Props, State> {
             this.fetchData(nextProps);
         } else if (!hasBeenToggled && hasVisibilityChanged) {
             this.setState({
-                view: this.getDefaultSidebarView(file, nextProps),
+                view: this.getDefaultSidebarView(nextProps, file, editors),
```

Comment:
Can this concept be removed from state and just be derived from props in `render`? It seems like we're going through a lot of trouble to pass `file` and `editors` around to make this work.